Upgrade to OpenEthereum v3.2.5

yucer-elbt opened the pull request:
According to configuration changes proposed in [EWF
Docs](https://energyweb.atlassian.net/wiki/spaces/EWF/pages/706478224/Updating+the+Client).

yucer-elbt committed:
Upgrade to OpenEthereum v3.2.5

According to configuration changes proposed in [EWF
Docs](https://energyweb.atlassian.net/wiki/spaces/EWF/pages/706478224/Updating+the+Client).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/energywebfoundation/ewf-rpc?pullRequest=4) <br/>All committers have signed the CLA.

yucer-elbt commented:
I just noted that the initial status of `disable_periodic` in the (instructions)[https://energyweb.atlassian.net/wiki/spaces/EWF/pages/706478224/Updating+the+Client] was:

`disable_periodic = true`

The setting was removed... and `enable=false` remains in its place with:

`enable = false`

but here in the *ewf-rpc* the setting was:

`disable_periodic = false`  

should the value of enable be `true` ?:

`enable = true` 

I mean is the new setting related with the old one ? Or are they just distinct settings?

royki commented:
> I just noted that the initial status of `disable_periodic` in the (instructions)[https://energyweb.atlassian.net/wiki/spaces/EWF/pages/706478224/Updating+the+Client] was:
> 
> `disable_periodic = true`
> 
> The setting was removed... and `enable=false` remains in its place with:
> 
> `enable = false`
> 
> but here in the _ewf-rpc_ the setting was:
> 
> `disable_periodic = false`
> 
> should the value of enable be `true` ?:
> 
> `enable = true`
> 
> I mean is the new setting related with the old one ? Or are they just distinct settings?

For Validator node, there is no need to have snapshot. In OpenEthereum , this is by default enabled now. That's why we made it false. And previous setup also it was disable via `disable_periodic`

For RPC snapshot is enabled. 
Sorry I didn't notice this. Good catch.
Please update this.

royki commented on volta-rpc/install-rpc-ubuntu-server-18.04-volta.bash:
```diff
@@ -205,14 +200,13 @@ warp = false
 
 [snapshots]
-disable_periodic = false
+enable = false
```
It should be `enable = true`

royki commented on ewc-rpc/install-rpc-ubuntu-server-18.04-ewc.bash:
```diff
@@ -205,14 +200,13 @@ warp = false
 
 [snapshots]
-disable_periodic = false
+enable = false
```
It should be `enable = true`

yucer-elbt committed:
[FIX] enabled = true

According to royki:

    For Validator node, there is no need to have snapshot. In
    OpenEthereum , this is by default enabled now. That's why we made
    it false. And previous setup also it was disable via
    disable_periodic

    For RPC snapshot is enabled.

yucer-elbt commented:
@royki 

I think it is fixed now.
